src/corona_hakab_model/generation/circles_consts.py

```diff
@@ -93,12 +93,30 @@ def get_age_distribution(self):
 
     def get_connection_types_prob_by_age(self):
         return {age: self.connection_type_prob_by_age_index[i] for i, age in enumerate(self.ages)}
-
+    
     # overriding hash and eq to allow caching while using un-hashable attributes
     __hash__ = object.__hash__
     __eq__ = object.__eq__
 
 
+    def get_required_adult_distributions(self):
+        #TODO might be other ages
+        teacher_student_ratio = ((self.age_prob[1]*self.connection_type_prob_by_age_index[1][ConnectionTypes.School] + 
```

Comment:
as you said, there are soon going to be more ages...
this can be done more generically by looping through the ages, and treating all people above a given age (let's say 20 for now) as teachers, and the rest as students.

Reply:
resolved